eng/pipelines/templates/steps/workload-publish.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  publishToFeed: false
+  feedForPublishing: public/dotnet10-workloads
```

Comment:
I think just defaulting this to empty string `''` so that someone doesn't think this needs to be maintained across branch changes. Only the value in the pipeline yml should be updated.

Reply:
Ohh missed that you're right :)